digdag-spi/src/main/java/io/digdag/spi/StorageFactory.java

```diff
@@ -1,10 +1,11 @@
 package io.digdag.spi;
 
 import io.digdag.client.config.Config;
+import io.digdag.client.config.ConfigException;
 
 public interface StorageFactory
 {
     String getType();
 
-    Storage newStorage(Config config);
+    Storage newStorage(Config config) throws ConfigException;
```

Comment:
Could you explain the reason change the SPI ?

Reply:
This is just an example. 
Suppose we tried to open the path of a file containing authentication information, but failed because the file does not exist.
 I think this error is due to a misconfiguration and should be returned to the caller.

Reply:
Sorry, I can't understand what your description correctly.
ConfigException inherits RuntimeException. So we can throw ConfigException without this.
In addition there is no implementation to process ConfigException in caller.
I guess it is not necessary for this PR, isn't it?

Reply:
Sorry, It was a misunderstanding.
It is not necessary for this PR.